packages/styles/scss/components/radio-button/_radio-button.scss

```
@@ -0,0 +1,196 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+//-----------------------------
+// Radio
+//-----------------------------
+
+@use '../../theme' as *;
+@use '../../type';
+@use '../../utilities/focus-outline' as *;
+@use '../../utilities/component-reset' as *;
+@use '../../utilities/visually-hidden' as *;
+@use '../../utilities/skeleton' as *;
+@use '../../config' as *;
+@use '../../spacing' as *;
+
```

```suggestion
@use '../form';
```

We'll need to pull in the label styles from `Form`